examples/notebooks/zeppelin/MNIST-kmeans.json

```diff
@@ -1 +1 @@
e Amazon SageMaker Endpoint for transformation.\n<br  />The ResponseRowDeserializer is what the SageMakerModel uses upon a call to transform() to the Amazon SageMaker Endpoint's response from a byte array into Rows of a DataFrame.</p>\n<p>The ProtobufRequestRowSerializer serializes Spark Rows into a protobuf format that the KMeans, PCA, and LinearLearner algorithms can consume for inference.\n<br  />The KMeansProtobufResponseRowDeserializer deserializes the protobuf response from the KMeans model image.</p>\n"}]},"apps":[],"jobName":"paragraph_1524718341814_-2141140640","id":"20171027-194607_461915056","dateCreated":"2018-04-26T04:52:21+0000","dateStarted":"2018-04-26T05:07:15+0000","dateFinished":"2018-04-26T05:07:15+0000","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:111"},{"text":"%spark\n\nimport com.amazonaws.services.sagemaker.sparksdk.SageMakerEstimator\nimport com.amazonaws.services.sagemaker.sparksdk.transformation.serializers.ProtobufRequestRowSerializer\nimport com.amazonaws.services.sagemaker.sparksdk.transformation.deserializers.KMeansProtobufResponseRowDeserializer\n\nval kmeans = new SageMakerEstimator(\n      trainingImage = \"382416733822.dkr.ecr.us-east-1.amazonaws.com/kmeans:1\",\n      modelImage = \"382416733822.dkr.ecr.us-east-1.amazonaws.com/kmeans:1\",\n      requestRowSerializer = new ProtobufRequestRowSerializer(),\n      responseRowDeserializer = new KMeansProtobufResponseRowDeserializer(),\n      hyperParameters = Map(\"k\" -> \"10\", \"feature_dim\" -> \"784\"),\n      sagemakerRole = IAMRole(roleArn),\n      trainingInstanceType = \"ml.p2.xlarge\",\n      trainingInstanceCount = 2,\n      endpointInstanceType = \"ml.c4.xlarge\",\n      endpointInitialInstanceCount = 1)","dateUpdated":"2018-04-26T05:49:03+0000","config":{"colWidth":12,"editorMode":"ace/mode/scala","results":{},"enabled":true,"editorSetting":{"language":"scala","editOnDblClick":false},"tableHide":true},"settings":{"params":{},"forms":{}},"apps":[],"jobName":"paragraph_1524718341814_-2141140640","id":"20171027-195104_2011298555","dateCreated":"2018-04-26T04:52:21+0000","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:112"},{"text":"%md\n\n\n","dateUpdated":"2018-04-26T05:48:38+0000","config":{"tableHide":false,"editorSetting":{"language":"markdown","editOnDblClick":true},"colWidth":12,"editorMode":"ace/mode/markdown","editorHide":true,"results":{},"enabled":true},"settings":{"params":{},"forms":{}},"apps":[],"jobName":"paragraph_1524718341814_-2141140640","id":"20171027-195210_612973143","dateCreated":"2018-04-26T04:52:21+0000","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:113"}],"name":"MNIST-kmeans","id":"2DBQGGZPN","angularObjects":{"2BRWU4WXC:shared_process":[],"2AM1YV5CU:shared_process":[],"2AJXGMUUJ:shared_process":[],"2ANGGHHMQ:shared_process":[],"2AKK3QQXU:shared_process":[]},"config":{"looknfeel":"default","personalizedMode":"false"},"info":{}}
```

Comment:
It is really difficult to see what has changed :(.

Reply:
Yeah, I described what changed in the description, but you'll have to describe . I'm looking at the diff and the README doesn't look like it'll be overridden -- could you elaborate?

Reply:
There used to be a commit, in the commit history. Not too sure what happened to it, maybe a force push? It isn't there anymore, so disregard my comment. Now you're making me wish that I took a picture of it.